Source/WebCore/html/HTMLTextFormControlElement.h

```diff
@@ -144,6 +144,7 @@ class HTMLTextFormControlElement : public HTMLFormControlElement {
     void internalSetMaxLength(int maxLength) { m_maxLength = maxLength; }
     void internalSetMinLength(int minLength) { m_minLength = minLength; }
 
+    bool isForBindings { false };
```

Comment:
Please don't add a new public boolean member variable like this.
Instead, we should be passing this argument across function calls.

Reply:
Adding a new argument to the function seems fine. Don't add a new state to HTMLTextFormControlElement, let alone a public state variable.